h/views/admin_groups.py

```diff
@@ -45,18 +45,36 @@ def get(self):
     @view_config(request_method='POST')
     def post(self):
         def on_success(appstruct):
-            read_url = self.request.route_url('admin_groups')
-            self.request.session.flash('TODO: I will add a {gtype} group called "{name}"'
-                                       ' for authority {authority}, created by {creator}'
-                                       ' and origins {origins}'.format(
-                                            gtype=appstruct['group_type'],
-                                            name=appstruct['name'],
-                                            authority=appstruct['authority'],
-                                            creator=appstruct['creator'],
-                                            origins=', '.join(appstruct['origins'])
-                                       ), queue='success')
-            response = HTTPFound(location=read_url)
-            return response
+            svc = self.request.find_service(name='group')
+
+            # Create the new group.
+            creator = appstruct['creator']
+            authority = appstruct['authority']
+            description = appstruct['description']
+            name = appstruct['name']
+            origins = appstruct['origins']
+            type_ = appstruct['group_type']
+
+            userid = 'acct:{}@{}'.format(creator, authority)
```

Comment:
I think it is slightly odd that the service's `create*` methods take a userid rather than username and authority, but I didn't want to change that in this PR.

Reply:
I agree that feels wrong, but would need to be fixed separately